exchanges/deribit/deribit_test.go

```diff
@@ -465,13 +468,11 @@ func TestGetLastTradesByCurrencyAndTime(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
-	_, err = d.WSRetriveLastTradesByCurrencyAndTime(currencyBTC, "", "", 0, false,
-		time.Now().Add(-8*time.Hour), time.Now())
+	_, err = d.WSRetriveLastTradesByCurrencyAndTime(currencyBTC, "", "", 0, false, time.Now().Add(-8*time.Hour), time.Now())
```

Comment:
These last 3 function calls do not return any data anymore. [Continued From](https://github.com/thrasher-corp/gocryptotrader/pull/1082#discussion_r1027500604)